bundler/spec/commands/newgem_spec.rb

```diff
@@ -175,11 +245,38 @@ def bundle_exec_rubocop
     end
   end
 
-  shared_examples_for "--no-rubocop flag" do
+  shared_examples_for "--linter=standard flag" do
+    before do
+      bundle "gem #{gem_name} --linter=standard"
+    end
+
+    it "generates a gem skeleton with standard" do
+      gem_skeleton_assertions
+      expect(bundled_app("test-gem/Rakefile")).to read_as(
+        include('require "standard/rake"').
+        and(match(/default:.+:standard/))
+      )
+    end
+
+    it "includes standard in generated Gemfile" do
+      allow(Bundler::SharedHelpers).to receive(:find_gemfile).and_return(bundled_app_gemfile)
+      builder = Bundler::Dsl.new
+      builder.eval_gemfile(bundled_app("#{gem_name}/Gemfile"))
+      builder.dependencies
+      standard_dep = builder.dependencies.find {|d| d.name == "standard" }
+      expect(standard_dep).not_to be_nil
+    end
+
+    it "generates a default .standard.yml" do
+      expect(bundled_app("#{gem_name}/.standard.yml")).to exist
+    end
+  end
+
+  shared_examples_for "--linter=none flag" do
```

Comment:
This is interesting. I think you probably used "none" here because the prompt says "rubocop/standard/(none)", right? I think "none" in the prompt is meant as "leave empty, just hit enter". I think `--linter=none` and in general `--linter=any_other_undefined_value` works fine accidentally because there's no behaviour associated to it.

Actually "none" reads slightly better to me, and we will never add a linter named "none" that would mess things up, but since `false` is already used for configuring through ENV (`BUNDLE_GEM__LINTER=false`), or through bundler config (`bundle config linter false`), I'd stick to `--linter=false` here?

By the way, thanks for testing this case, I don't think it's tested for other similar flags.

Reply:
Yeah, I used "none" just because it reads nicer. Both this and the ci flag will take non-"false" flags and parse them as strings, and then when they aren't present in the `case` statement, they'll ultimately resolve to false. Seems fine?

Reply:
Sure, let's leave it as it is.